app/views/govuk_publishing_components/components/_phase_banner.html.erb

```diff
@@ -13,10 +14,14 @@ unless message.present?
   elsif phase == "alpha"
     message = raw("This part of GOV.UK is being built &ndash; <a href=\"/service-manual/phases/ideal-alphas\">find out what alpha means</a>")
   end
-end 
+end
+
+container_css_classes = ["gem-c-phase-banner",  "govuk-phase-banner"]
```

Comment:
I think you can do this as `container_css_classes = %w(gem-c-phase-banner govuk-phase-banner)`, which means you don't need to do the `join(' ')` on line 24.

Reply:
Nice - thanks for the tip!